### DISCUSSION TITLE
rlxos Silaghana

### DISCUSSION THREAD

**Discussion by itsManjeet** [itsManjeet/avyos#70] [Announcements]
Exciting news! 🎉 We've just unleashed RLXOS Silaghana, a groundbreaking release that's all about rock-solid stability and elevated performance.

This update brings a whole new level of reliability to your computing experience. We've turbocharged the desktop environment, made updates seamless with Swupd, widened compatibility across devices, and refined overall system stability.

But wait, there's more! We've detailed all these fantastic updates and enhancements in a fresh blog post waiting for you to explore. 🚀 Dive in to discover how RLXOS Silaghana can take your user experience to new heights!

Check out the [blog post](https://rlxos.dev/blog/rlxos-silaghana/) to learn about the improvements, new features, and how they'll impact your day-to-day usage.

Your feedback matters! Join the conversation, share your thoughts, and tell us about your experiences with RLXOS Silaghana. Your insights guide our journey toward continuous improvement.

Get ready to experience stability like never before with RLXOS Silaghana. Read the blog post and let's start this discussion together! 💬

Cheers,

**Comment by itsManjeet** [comment]
Your feedback matters! Join the conversation, share your thoughts, and tell us about your experiences with RLXOS Silaghana. We're eager to hear about any bugs or missing features you encounter. Your insights guide our journey toward continuous improvement.

**Reply by ghost** [reply]
Already Known Issues
- [x] Docker take a extra delay of 20 - 30 sec after login to start
- [x] Some packages are not configured for properly use provided ca-certificates
- [x] `no suitable ... mode, booting into blind mode` grub.cfg
- [x] Initial-setup is ugly 💦

**Reply by itsManjeet** [reply]
FIXED with c756ec5c88d2402c0ada592975a34e98dd3a9c82
